src/ddmd/mtype.d

```diff
@@ -3075,6 +3075,13 @@ extern (C++) abstract class Type : RootObject
     {
         v.visit(this);
     }
+
+    final TypeFunction toTypeFunction()
+    {
+        if (ty != Tfunction)
+            assert(0);
```

Comment:
> It'll be useful to someone debugging a problem in the compiler.

I've been debugging the compiler for 30 years, and such information adds nothing to what the source code and comments around the assert tells you.

> atleast have something to put into bugzilla

There is - an assert message giving file and line.